modules/cluster-logging-manual-rollout-rolling.adoc

```diff
@@ -12,22 +12,23 @@ Also, a rolling restart is recommended if the nodes on which an Elasticsearch po
 
 .Prerequisites
 
-* OpenShift Logging and Elasticsearch must be installed.
+* The Red Hat OpenShift Logging and Elasticsearch Operators must be installed.
```

Comment:
clusterlogging/instance in Unmanage status
elasticsearch/elasticsearch in Unmanage status